docs/CHANGELOG.md

```diff
@@ -3,6 +3,10 @@ Version 3.14.0 (June 1, 2021)
 
 **User-visible changes:**
 
+The Resource Leak Checker ensures that certain methods are called on an
+object before it is de-allocated. By default, it enforces that `close()` is
+called on any expression whose type might be `java.io.Closeable`.
```

Comment:
"type might be Closeable" implies that "type" is run-time type, because the compile-time type is always known exactly.
If you mean compile-time type, I would say "compile-time type is a subtype of Closeable" or the like.